README.md

```diff
@@ -4,47 +4,97 @@
 uftrace
 =======
 
-The uftrace tool is to trace and analyze execution of a program
-written in C/C++.  It was heavily inspired by the ftrace framework
-of the Linux kernel (especially function graph tracer) and supports
-userspace programs.  It supports various kind of commands and filters
-to help analysis of the program execution and performance.
+uftrace is a function call graph tracer for C, C++, Rust and Python programs.
+Using dynamic tracing, it can include nested library calls and
+even seamless function graph tracing of used kernel functions.
+
+- Shows colored nested function call graphs (instead of listings)!
+- Displays arguments symbolically using libc function prototypes and DWARF
+  debug information values.
+- Filters like minimal function call duration are available
+- Traces trace kernel events e.g. scheduling events (which affecting the
+  execution timing of the program) and records nanosecond-exact timestamps.
+
+uftrace traces, records and analyzes the execution of programs written in
+C, C++ and Rust. It was heavily inspired by the ftrace framework of the
+Linux kernel (especially the function graph tracer) and supports userspace
+programs, and with elevated privileges, also kernel function call graphs.
+
+The name uftrace stems from the combination of user and ftrace.
+
+It supports many commands and filters such as filtering by function call
+duration for analysis of program execution and performance.
 
 ![uftrace-live-demo](doc/uftrace-live-demo.gif)
 
  * Homepage: https://github.com/namhyung/uftrace
  * Tutorial: https://github.com/namhyung/uftrace/wiki/Tutorial
  * Chat: https://gitter.im/uftrace/uftrace
  * Mailing list: [uftrace@googlegroups.com](https://groups.google.com/forum/#!forum/uftrace)
+ * Lightning talk: https://youtu.be/LNav5qvyK7I
 
 
 Features
 ========
 
-It traces each function in the executable and shows time duration.  It
-can also trace external library calls - but usually entry and exit are
-supported.  Optionally it's possible to trace other (nested) external
-library calls and/or internal function calls in the library call.
-
-It can show detailed execution flow at function level, and report which
-function has the highest overhead.  And it also shows various information
-related the execution environment.
+uftrace traces each function in the executable and shows time durations.
+
+Usually, for this to be possible, the program needs to be compiled with
+`-pg` or `-fpatchable-function-entry=2`, but with fully dynamic tracing
+(`-P.`|`--patch=.`), uftrace works on all executables (as long they are
+not stripped, or symbol information is available from a separate file).
+
+uftrace is able to perform nested library call tracing even on stripped
+executables with no symbol information using (`-l`|`--nest-libcall`)
+by hooking into the procedure linkage tables of shared libraries.
```

Comment:
This description is only for the first level library function tracing. To explain about `--nest-libcall`, we better explain uftrace hooks PLTs inside libraries as well as the given executable file.

Reply:
It's the `--force` (or `-P.`) option which enables (flat) library call tracing on stripped binaries.

Reply:
Yes, I was actually mention hooking into the PLT (jus using "procedure linkage tables"), added "PLT" now as well.

Thanks @namhyung for correcting the info Honggyu Kim, and I'd like to add that -D1 is more clear for flat/no-nesting. I've added this to the paragraph.